new/rfcXXX-lifecycle-and-release-policies.md

```diff
@@ -0,0 +1,214 @@
+---
+RFC: unassigned
+Title: Chef OSS Lifecycle and Release Policies
+Author: Tom Duffield
```

Comment:
By convention should have an email address here, though maybe it should be a list address for the maintainers so you don't get spam/complaints :)